CFC_DataCollector/trip_analysis/featurization.py

```diff
@@ -122,7 +134,7 @@ def map_clusters(self):
             end_lat = self.data[i]['trip_end_location'][1]
             end_lon = self.data[i]['trip_end_location'][0]
             path = [(start_lat, start_lon), (end_lat, end_lon)]
-            mymap.addpath(path, matcol.rgb2hex(colormap(float(self.colors[i])/self.clusters)))
+            mymap.addpath(path, matcol.rgb2hex(colormap(float(self.colors[i])/len(set(self.colors)))))
```

Comment:
Previous comment about reusing existing function to plot trips and sections using existing code.